routes/api/contacts.js

```diff
@@ -1,25 +1,102 @@
-const express = require('express')
+const express = require("express");
+const router = express.Router();
+const createError = require("http-errors");
+const Joi = require("joi");
 
-const router = express.Router()
+const contactSchema = Joi.object({
+  name: Joi.string().required(),
```

Comment:
Joi schema should be in separate file called schema.js

Reply:
Done. Check, please.